packages/frontend/web-ui/src/common/helpers/isSerializableAppError.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { beforeAll, describe, expect, it } from '@jest/globals';
+
+import { AppErrorKind } from '@cornie-js/frontend-common';
+
+import { isSerializableAppError } from './isSerializableAppError';
+
+describe(isSerializableAppError.name, () => {
+  describe.each<[string, unknown, boolean]>([
+    [
+      'a SerializableAppError',
+      {
+        kind: AppErrorKind.contractViolation,
+        message: 'message-fixture',
+      },
+      true,
+    ],
+    [
+      'a SerializedError',
```

Comment:
This case is too specific. We are writing an unit test here. I'm missing the following cases:
- A non object.
- `null` value.
- An object with no `kind`.
- An object with invalid `kind`.
- An object with valid kind and no message.

Reply:
I've deleted the case of SerializedError and I've added the cases that you commented.